src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -68,9 +70,14 @@ export class HuiDialogEditCard extends LitElement {
         `ui.panel.lovelace.editor.card.${this._cardConfig.type}.name`
       )} ${this.hass!.localize("ui.panel.lovelace.editor.edit_card.header")}`;
     } else {
-      heading = this.hass!.localize(
-        "ui.panel.lovelace.editor.edit_card.header"
-      );
+      heading =
```

Comment:
This is now changed for every card editor, not only the card picker? You should check if _cardConfig is undefined

Reply:
So instead of else another if, checking if _cardConfig === undefined?

Reply:
Implemented check for undefined _cardConfig in f5d3aa0e830edd5130ecc2a5e5c534b9f135304a